packages/ignition/src/mdx-components.tsx

```diff
@@ -67,7 +66,7 @@ export function useMDXComponents(components: MDXComponents): MDXComponents {
     ),
     code: ({ children, ...props }: CodeProps) => (
       <CodeContainer {...props}>
-        <Code>{children}</Code>
+        <Block>{children}</Block>
```

Comment:
In a future update, I think this should be called CodeBlock to match the parent naming style "CodeContainer"

Reply:
Sure